presto-raptor-legacy/src/main/java/io/prestosql/plugin/raptor/legacy/storage/OrcFileWriter.java

```diff
@@ -92,7 +92,7 @@
         }
     }
 
-    private static final Configuration CONFIGURATION = new Configuration();
+    private static final Configuration CONFIGURATION = new Configuration(true);
```

Comment:
was intention here really to read defaults or just to have configuration for constructor?

Reply:
Good question. However, even if was not intentional, IMO we're no longer in position to change it.
@electrum thoughts?

BTW there are in total 3 cases like this in this PR.

Reply:
This was not intended to load defaults and should be fixed.